test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -34,7 +34,7 @@
 #include "model/permissions.hpp"
 #include "module/irohad/ametsuchi/ametsuchi_fixture.hpp"
 
-#include "ametsuchi/impl/redis_flat_block_query.hpp"
+#include "ametsuchi/impl/redis_block_query.hpp"
```

Comment:
Does this include is really required? If it changed and nothing changed in file that means yes